content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,126 @@
+{/* First cut from integrations team */}
+### Sunsetting of the Sourcegraph extensions
+
+We will be deprecating our current Sourcegraph extensions framework so that we can make way for a new model that allows a deeper integration throughout our entire code intelligence platform.
+
+In this release your favorite extensions (git-extras, open-in-editor and search-exports) will become part of the core application. Making them faster, more discoverable and ensuring they continue to be improved upon.
+
+### Promoting usage of Browser and IDE extensions
+
+We've added a page to our admin analytics to help you understand the adoption and amount of time saved from using IDE and Browser extensions across your organization. 
+
+To make it easier to roll out our VS Code extension organization wide, we've created an example batch change that adds the extension to your repository’s recommended extensions. Once added, users will be recommended the Sourcegraph extension when opening the repository if they don't have it installed already. 
+
+
+{/* First cut from Repo/IAM/Admin Analytics team */}
+
+### Complete Admin Analytics
+
+With 3.42, we introduced an all-new admin analytics including. Now with 4.0, we're significantly improving on the initial release. 4.0 includes dozens of updates including pages for all Sourcegraph features, annual projections for dev-time-saved, data exports, individual user usage table, and much more. 
+
+### 1-click log export 
+
+We would love if everything always worked with Sourcegraph, but no matter how much you test, bugs will always arise. With the introduction of 1-click log exports, we're making it simple to export all logs associated with Sourcegraph to ease the debugging process. 
+
+### Repo config Experience (such as Repo syncing status)
+
+Configuring code host connections and permissions syncing is a core prat of the admin experience, and historically, it has been painful to do. With Sourcegraph 4.0, we have introduced more than X quality of life improvements to help admins understand the current state of repository and permissions syncing. 
+
+{/* @benvenker Search Product and core workflow draftt */}
+### Smarter code search
+Search is now more intelligent by defaul and automtically generates and runs alternative queries when your initial query doesn't have many results. This can be disabled so you still get only literal or structural matches when you need to, but in many cases the new smarter search is likely to be better. This will especially benefit new users of Sourcegraph who don't yet know the query syntax. The default query syntax is also simplified, allowing you to combine literal search terms and regular expressions with `/`'s like so: `authentication /^my|regex$/`.
+
+### Simplified UI
+We've made a number of small usabilty improvements to simplify the search UI and make Sourcegraph feel less complex.
```

Comment:
```suggestion
We've made a number of usabilty improvements to simplify the search UI and make Sourcegraph feel more concise and easier to use.
```

Thinking that this is a bunch of small changes that make a big impact as a whole, so we shouldn't undercut it. I'm also suggesting phrasing it so that instead of "perceived bad" → "less bad," it's solely "better!"